web/packages/teleport/src/Databases/fixtures/index.ts

```diff
@@ -30,7 +30,10 @@ export const databases: Database[] = [
       { name: 'env', value: 'aws' },
     ],
     hostname: 'aurora-hostname',
-  },
+    targetHealth: { status: 'unhealthy' },
+    // TODO(kimlisa): remove as any once we add
+    // "targetHealth" field to db response
+  } as any,
```

Comment:
i know we don't like temporary stuff in our code base but i thought it was okay for the sake of story (i will have a follow up PR that removes the `as any`)